test/blackbox-tests/test-cases/include-qualified/data-only-dirs-malformed.t/run.t

```diff
@@ -0,0 +1,6 @@
+The presence of a directory whose name is not a valid module name should not
+clash with (include_subdirs qualified) if the directory in question is ignored
+by a `(dirs :standard \ ...)` stanza.
```

Comment:
the description in some of these tests doesn't match what's being tested. You mention using `(dirs..)` but then the test uses `data_only_dirs`.